HW1/P3/P3.py

```diff
@@ -0,0 +1,19 @@
+import findspark
+findspark.init()
+import pyspark
+
+def main():
+    sc = pyspark.SparkContext()
+    wlist = sc.wholeTextFiles("./CSV Format")
+    combined_wlist = wlist.flatMap( lambda (a,b): b.split() )
+    combined_wlist_sort = combined_wlist.map( lambda x: ( "".join( sorted( x ) ), [x] ) )
+    combined_wlist_sort_reduce = combined_wlist_sort.reduceByKey( lambda x,y: x + y )
+    combined_wlist_sort_reduce_count = combined_wlist_sort_reduce.map( lambda (x,y): (x, [y]) if ( type(y) is not list ) else (x, 
```

Comment:
When there isnt a common key to join on, y would simply be NULL. I was handling that case